src/knot_cloud.c

```diff
@@ -407,9 +361,7 @@ int knot_cloud_register_device(const char *id, const char *name)
 		return KNOT_ERR_CLOUD_FAILURE;
 
 	json_str = json_object_to_json_string(jobj_device);
-
-	headers[0].value.value.bytes = amqp_cstring_bytes(user_auth_token);
-
+	
```

Comment:
It would be interesting to just leave the line as it was

Reply:
> I've made some change requests in respect to comments and blank lines, but the code looks good!
> If it is possible, I would just consider splitting this commit in more than one commit.
> E.g: One to effectively decouple the knot_cloud from amqp calls, another one to Removes unused queue_reply;
> Great Pull Request!

I agree with you, i will do this